src/components/Tabs/TabsNav.test.tsx

```diff
@@ -27,30 +28,101 @@ afterEach(() => {
   jest.clearAllMocks();
 });
 
-const renderComponent = (props?: TabsNavProps) => render(<TabsNav {...DEFAULT_PROPS} {...props} />);
+describe('<TabsNav />', () => {
+  describe('class names', () => {
+    test('should apply Container class only to the container element', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const tabNavContainer = container.getElementsByClassName(styles.Container);
 
-test('should render react-router-dom Link elements with correct class names', () => {
-  const { getAllByTestId } = renderComponent();
-  const links = getAllByTestId('link');
-  expect(links.length).toEqual(MOCK_TABS.length);
-  links.forEach(l => expect(l.classList).toContain('Tab'));
-});
+      expect(tabNavContainer.length).toBe(1);
+    });
 
-test('should apply Selected class name on selected Link', () => {
-  const { container } = renderComponent();
-  const selected = container.getElementsByClassName('Selected');
-  expect(selected.length).toBe(1);
-  expect(selected[0].textContent).toBe(MOCK_TABS[0].text);
-});
+    test('should apply List class only to the List element', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const list = container.getElementsByClassName(styles.List);
```

Comment:
This one checks that there's an element with the `styles.List` class name, but we can add more confidence by checking the text.
```js
expect(list[0].textContent).toBe(...);
```
Otherwise, `styles.List` could be applied to any other element and the test would pass.

Reply:
Updated, and the other tests like this :)